.github/workflows/sql-test-and-build-workflow.yml

```diff
@@ -2,6 +2,9 @@ name: SQL Java CI
 
 on: [push, pull_request]
 
+env:
+  CODECOV_TOKEN: 9be2b15e-5d49-4321-93ee-43528965467a
```

Comment:
are we ok to expose the token to public?

Reply:
I was not sure either at first- when I spoke to Yaliang he said that the community users can still download our code and build to generate the test coverage report, so since the code is public, the token isn't relevant for security protection.

What's your thoughts on this @dai-chen @penghuo ?